### PR TITLE
py_stringmatching 0.4.5 :snowflake:

### PR DESCRIPTION
**Pull request by ifitchet**
py_stringmatching 0.4.5 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5066]
- dev_url:        https://github.com/anhaidgroup/py_stringmatching/tree/v0.4.5
- conda_forge:    https://github.com/conda-forge/py_stringmatching-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/py_stringmatching/0.4.5
- pypi inspector: https://inspector.pypi.io/project/py_stringmatching/0.4.5

### Explanation of changes:

- basic build ignoring conda-forge restriction to `x86_64`
- add upstream tests

[PKG-5066]: https://anaconda.atlassian.net/browse/PKG-5066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 
 package:
-  name: py_stringmatching
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/py_stringmatching/py_stringmatching-{{ version }}.tar.gz
-  sha256: 921b9bb163b310df341c33823913204f7b0ca9f6a319f5b451408300bd378dee
+  url: https://pypi.io/packages/source/p/{{ name }}/py-stringmatching-{{ version }}.tar.gz
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py-stringmatching-{{ version }}.tar.gz
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+  sha256: d97ba1b562ad98effc946687c5dab074c17a515114ab0dcf502202d5cec9f2b5
 
 build:
-  number: 1
-  skip: true  # [not x86_64]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  #skip: true  # [not x86_64]
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
```

**Review by lorepirri — APPROVED** [review]
LGTM, a couple of nitpicks

File: recipe/meta.yaml (hunk 1 above)
Comment (ifitchet) [reply recipe/meta.yaml]: Hah!  Added that in the code 😉

File: recipe/meta.yaml (hunk 2 above)
Comment (ifitchet) [reply recipe/meta.yaml]: Yep, saw that waiting for the last build to complete before marking for review.  Then forgot to push the change.

**Review by ViridianMelody — APPROVED** [review]

**Pull request merged**